content/blog/2017-11-16-ropensci-changes.md

```diff
@@ -0,0 +1,104 @@
+
+Here is an overview of some of the ways we’ve made simple version control easy with changes:
+
+##### Simple terminology
+changes uses simple and deliberately un-git-like terminology:
+
+  * You start a new version control project with `create_repo()`, which is like `git init` but it can set up a nice project directory structure for you, automatically ignoring things like output folders.
+  * All of the steps involved in commiting edits have been compressed into one function: `record()`. All files that aren't ignored will be committed, so users don't need to know the difference between tracking, staging and committing files. 
+  * It's easy to set which files to omit from version control with `ignore()`, and to change your mind with `unignore()`. 
+  * `changes()` lets you know which files have changed since the last record, like a hybrid of `git status` and `git diff`.
+  * You can look back in history with `timeline()` (a simplified version of `git log`), `go_to()` a previous record (like `git checkout`), and `scrub()` any unwanted changes since the last record (like `git reset --hard`).
+
+##### It's linear
+After an long discussion, we decided that changes won't provide an interface to Git branches (at least not yet), as the merge conflicts it leads to are one of the scariest things about version control for beginners.
+With linear version control, users can can easily `go_to()` a past record with a version number, rather than unfamiliar [shas](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects). These numbers appear in the a lovely visual representation of their `timeline()`:
+
+          (1) initial commit
+           |  2017-11-18 02:55
+           |
+          (2) set up project structure
+           |  2017-11-18 02:55
+           |
+          (3) added stuff to readme
+              2017-11-18 02:55
+
+If you want to roll your project back to a previous record, you can `retrieve()` it, and changes will simply append that record at the top of your timeline (storing all the later records, just in case).
+
+##### Readable messages and automatic reminders
+Some of Git's messages and helpfiles are totally cryptic to all but the most hardened computer scientists.
+Having been confronted with our fair share of [`detached HEAD`](https://www.git-tower.com/learn/git/faq/detached-head-when-checkout-commit)s and offers to [`update remote refs along with associated objects`](https://git-scm.com/docs/git-push), we were keen to make sure all the error messages and helpfiles in changes are as intuitive and understandable as possible.
+
+It can also be hard to get into the swing of recording edits, so changes will give you reminders to encourage you to use `record()` regularly. You can change the time interval for reminders, or switch them off, using `remind_me()`. 
+
+####
```

Comment:
this `####` shows up as text, as is. Were you trying for a horizontal line or other formatting?

Reply:
wow, all of these errors are things I added!

I was aiming for an empty heading, so the subsequent section didn't appear as part of the subsection above.

I think this can just be deleted